storefront/app/models/workarea/checkout.decorator

```diff
@@ -0,0 +1,7 @@
+module Workarea
+  decorate Checkout, with: :storefront_api do
+    def update(params = {})
+      steps.all? { |s| s.new(self).update(params) }
```

Comment:
I thought we discussed fixing this in base?

Reply:
Also this has the problem you brought up where it will stop evaluating when it's the first `false`